wagtail/snippets/templates/wagtailsnippets/snippets/revisions/_actions.html

```diff
@@ -1,36 +0,0 @@
-{% load i18n wagtailadmin_tags %}
-
-<td {% if column.classname %}class="{{ column.classname }}"{% endif %}>
-    {% if revision_enabled and instance.revision %}
-        {% with revision=instance.revision latest_revision_id=object.latest_revision_id previous_revision_id=instance.previous_revision_id %}
-            <div class="w-flex w-items-center w-gap-2">
-                {{ value }}
-                {% if draftstate_enabled %}
-                    {% if instance.action == 'wagtail.publish' and revision.pk == object.live_revision_id %}{% trans 'Live version' as status_label %}{% status status_label classname="w-status--primary" %}
-                    {% elif instance.content_changed and revision.pk == latest_revision_id %}{% trans 'Current draft' as status_label %}{% status status_label classname="w-status--primary" %}{% endif %}
-                {% endif %}
-            </div>
-            <ul class="actions">
-                {% if preview_enabled and object.is_previewable %}
-                    <li><a href="{% url view.revisions_view_url_name object.pk|admin_urlquote revision.pk %}" class="button button-small button-secondary" target="_blank" rel="noreferrer">{% trans 'Preview' %}</a></li>
-                {% endif %}
-                {% if revision.pk == latest_revision_id %}
-                    <li><a href="{% url view.edit_url_name object.pk|admin_urlquote %}" class="button button-small button-secondary">{% trans 'Edit' %}</a></li>
-                {% else %}
-                    <li><a href="{% url view.revisions_revert_url_name object.pk revision.pk %}" class="button button-small button-secondary">{% trans 'Review this version' %}</a></li>
-                {% endif %}
-                {% if previous_revision_id %}
-                    <li><a href="{% url view.revisions_compare_url_name object.pk previous_revision_id revision.pk %}" class="button button-small button-secondary">{% trans 'Compare with previous version' %}</a></li>
-                {% endif %}
-                {% if revision.pk != latest_revision_id %}
-                    <li><a href="{% url view.revisions_compare_url_name object.pk revision.pk 'latest' %}" class="button button-small button-secondary">{% trans 'Compare with current version' %}</a></li>
-                {% endif %}
-                {% if can_publish and revision.approved_go_live_at %}
-                    <li><a href="{% url view.revisions_unschedule_url_name object.pk revision.pk %}" class="button button-small button-secondary">{% trans 'Cancel scheduled publish' %}</a></li>
-                {% endif %}
```

Comment:
The old logic in here didn't check whether the log entry has `content_changed = True`, because I thought "it will have a different revision instance anyway, so why not just give the action options?". However, I didn't realise that this could cause confusion because you'll be given these options but they may not always be useful (as is the case when the content didn't change).